Add tessa's changes, clean up comments/signposting, missing citations, and add abstract (#10)
taylorreiter/2021-paper-metapangenomes: content/06.methods.md

# Methods 

All code is available at github.com/dib-lab/2021-panmers (results section 1), github.com/dib-lab/2021-orpheum-sim (results section 2), and https://github.com/dib-lab/2021-metapangenome-example (results section 3).

CTB: please provide zenodo DOIs :)

### Selection of benchmarking species for pangenome analysis
We selected a species representative for each of the 23 phyla in GTDB rs202 [@doi:10.1093/nar/gkab776]. 

> To select representative species, we first filtered species with fewer than 20 genomes and greater than 1000 genomes.

Reviewer: ```suggestion
To select representative species, we first removed species with fewer than 20 genomes and greater than 1000 genomes.
```

### Determining short read open reading frames with orpheum 
We used the orpheum tool to predict open reading frames from simulated short reads [@doi:10.1101/2021.07.09.450799].
Orpheum was developed to predict open reading frames in short RNA-seq reads from Eukaryotic organisms without a reference genome or transcriptome sequence [@doi:10.1101/2021.07.09.450799].

> Orpheum perform six-frame translation on nucleotide sequencing reads, calculates k-mers in an amino acid encoding at the designated k-mer length, and then estimates the Jaccard similarity between k-mers in each translation frame and a database. 

Reviewer 2: Containment (of ref k-mers in each translated frame)

Author: ```suggestion
Orpheum performs six-frame translation on nucleotide sequencing reads, calculates k-mers in an amino acid encoding at the designated k-mer length, and then estimates the containment of k-mers in a reference database in each translated frame. 
```

Using quality controlled reads from each sample, we assembled each metagenome separately using megahit with default parameters [@doi:10.1093/bioinformatics/btv033].
We binned the resultant assemblies using metabat2 with parameter `-m 1500` [@doi:10.7717/peerj.7359].
We assigned GTDB species to each bin using sourmash `gather` (DNA, *k* = 31, scaled = 2000) against the GTDB rs202 database, selecting the species of the best match [@doi:10.1101/2022.01.11.475838].

> We decontaminated each bin using charcoal using default parameters [@url:https://github.com/dib-lab/charcoal].

Reviewer: ```suggestion
We decontaminated each bin with charcoal using default parameters [@url:https://github.com/dib-lab/charcoal].
```
